logistechMOTHAFUCKA/logistechMOTHAFUCKA.ino

```diff
@@ -141,47 +189,13 @@ void loop() {
       Serial.println(new_code, HEX);
       Serial.print("Total unique codes: ");
       Serial.println(num_codes);
```

Comment:
@princhcanal  can you help figure out why every time this condition is met, the value of `num_codes `is a random number 
![image](https://github.com/citu-cpe/logistech/assets/57755993/8f8a7496-4518-4020-bd5b-e55c726b4eea)